api/src/devcomp/domain/models/element/Embed-for-answer-verification.js

```diff
@@ -38,7 +38,7 @@ class EmbedForAnswerVerification extends Embed {
 
   #validateUserResponseFormat(userResponse) {
     const embedSchema = Joi.string()
-      .pattern(/^[a-z]+$/)
+      .pattern(/^[a-z.-]+$/)
```

Comment:
question : le point est nécessaire ?
```suggestion
      .pattern(/^[a-z-]+$/)
```

Reply:
Bien vu, pas besoin. Merci :)